### ISSUE TITLE
VCPU shutdown request on Fedora Core 5 and 16 x86 and x86_64

### ISSUE THREAD

**Issue by soul4soul** [intel/haxm#209]
**Describe the Bug**

I am running HAXM and QEMU on 6 different machines. Each machine has a different CPU and MOBO. On 5 of those 6 machines my VMs run wonderfully without issue. The 6th machine can't run any VMs with HAXM accel enabled.

Running Fedora 16 x86 VM fails to start with the message appearing every 500ms~ "VCPU shutdown request". 
Fedora Core 5 x86 fails in the same way with the same error message. Fedora 16 x86_64 fails in the same way with the same error message.

Sometimes the process dies immediately instead of printing "VCPU shutdown request" over and over.

**Host Environment**

* HAXM version: 7.4.1(Same issue with 7.5.1)

* Host OS version: Windows 10 Pro version 1803

* Host OS architecture: x86_64 (64-bit)

* Host CPU model: i7-6700HQ

* Host RAM size: 8GB

* (Optional) Host computer model: Lenovo Ideapad Y700

**Guest Environment**

* Android Emulator or QEMU version: QEMU 3.1.0

* Guest OS version and architecture: Fedora 16 x86, Fedora 16 x86_64, and Fedora Core 5 x86

**To Reproduce**

- `Start-Process -FilePath "C:\Program Files\qemu\qemu-system-x86_64.exe" -ArgumentList "-cpu qemu64 -smp cores=2 -device e1000,netdev=net0 -boot menu=off -m 1G -drive "file=C:\qimages\Fedora-16-x64.qcow2,index=0,media=disk" -snapshot -netdev user,id=net0,hostfwd=tcp::10022-:22 -accel hax" -WindowStyle Minimized -PassThru`
- `Start-Process -FilePath "C:\Program Files\qemu\qemu-system-i386.exe" -ArgumentList "-cpu qemu32 -smp cores=2 -device e1000,netdev=net0 -boot menu=off -m 1G -drive "file=C:\qimages\Fedora-16-x86.qcow2,index=0,media=disk" -snapshot -netdev user,id=net0,hostfwd=tcp::10022-:22 -accel hax" -WindowStyle Minimized -PassThru`
- `Start-Process -FilePath "C:\Program Files\qemu\qemu-system-i386.exe" -ArgumentList "-cpu qemu32 -smp cores=2 -device e1000,netdev=net0 -boot menu=off -m 1G -drive "file=C:\qimages\Fedora-Core-5-x86.qcow2,index=0,media=disk" -snapshot -netdev user,id=net0,hostfwd=tcp::10022-:22 -accel hax" -WindowStyle Minimized -PassThru`

**Expected Behavior**

Expectation: VM starts up without issue.

**Reproducibility**

100%

**Diagnostic Information**

HAXM log: 
[fedora-16-x86_64_dbgview.LOG](https://github.com/intel/haxm/files/3237509/fedora-16-x86_64_dbgview.LOG)
[fedora-16-x86_dbgview.LOG](https://github.com/intel/haxm/files/3237510/fedora-16-x86_dbgview.LOG)
[fedora-core-5-x86_dbgview.LOG](https://github.com/intel/haxm/files/3237512/fedora-core-5-x86_dbgview.LOG)

**Comment by coxuintel** [comment]
We'll check in our environment and share the feedback soon.

**Comment by HaHoYou** [comment]
From your log, this issue is similar with https://github.com/intel/haxm/issues/205, to comfirm that, either change to 1 core to launch, or change to low version of qemu 3.0.0, or just check the other machine's qemu version. 
We also tried to reproduce it, but our fedora is good to launch on Intel Skylake machine. For further trying, would you share with your qcow2 image file?

**Comment by soul4soul** [comment]
Changing to QEMU v3.0.0 and using HAXM 7.5.1 fixed the issue with no side effects. 

Changing the core count to 1 worked but I was only able to launch 1 VM at a time. Starting a second VM would fail. This is the output of debugview when starting a second VM.
```
00000634	2.96198034	haxm_error:	
00000635	2.96198082	...........hax_teardown_vm	
```

For now I will continue to use QEMU 3.0.0 and HAXM 7.5.1.

**Comment by HaHoYou** [comment]
Duplicate of #205

**Comment by HaHoYou** [comment]
The fix patch has been upstreamed to QEMU: 6f38dca615286796df9a967117f3ce42b918c8db. FYI, if you need higher version of qemu.
